tests/pytorch/nightly/common.libsonnet

```diff
@@ -38,13 +38,15 @@ local volumes = import 'templates/volumes.libsonnet';
   Convergence:: mixins.Convergence,
   PyTorchTpuVmMixin:: experimental.PyTorchTpuVmMixin {
     tpuSettings+: {
+      softwareVersion: 'tpu-vm-base',
       tpuVmPytorchSetup: |||
-        sudo pip3 uninstall --yes torch torch_xla torchvision numpy
-        sudo pip3 install https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch-nightly-cp38-cp38-linux_x86_64.whl https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch_xla-nightly-cp38-cp38-linux_x86_64.whl https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torchvision-nightly-cp38-cp38-linux_x86_64.whl numpy
+        sudo pip3 uninstall --yes torch torch_xla torchvision libtpu-nightly numpy
+        sudo pip3 install https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch-nightly-cp38-cp38-linux_x86_64.whl https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torchvision-nightly-cp38-cp38-linux_x86_64.whl https://storage.googleapis.com/tpu-pytorch/wheels/tpuvm/torch_xla-nightly-cp38-cp38-linux_x86_64.whl numpy
+        sudo pip3 install torch_xla[tpuvm]
         sudo pip3 install mkl mkl-include
         sudo apt-get -y update
         sudo apt-get install -y libomp5
-        git clone https://github.com/pytorch/pytorch.git
```

Comment:
why did we switch to only creating pytorch dir instead of clone?

Reply:
AFAIK none of the tests actually use anything in the PyTorch repository, but I created the directory anyway to keep the paths compatible. This is kind of obscure though, so I'll add a comment